### PR TITLE
Feature export Harbor statistics as Prometheus metric

### PR DESCRIPTION
**Pull request by tpoxa**
This PR exposes the Harbor statistics as Prometheus metric.

The data exposed as metrics are 1:1 taken from the`statistics` API, including the name.

## New Metrics

- harbor_statistics_private_project_amount
- harbor_statistics_private_repo_amount
- harbor_statistics_public_project_amount
- harbor_statistics_public_repo_amount
- harbor_statistics_total_project_amount
- harbor_statistics_total_repo_amount
- harbor_statistics_total_storage_consumption

Please indicate you've done the following:
- [x] Well-Written Title and Summary of the PR
- [x] Label the PR as needed.  **release-note/enhancement**
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [x] Made sure tests are passing and test coverage is added if needed.
- [x] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Comment by wy65701436** [comment]
@Vad1mo since we've reached the FC of v2.10 this week, I just put it into the v2.11 candidate.

File: src/pkg/exporter/project_collector_test.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package exporter
 
 import (
+	"github.com/goharbor/harbor/src/common/utils/test"
```
Comment (wy65701436) [inline src/pkg/exporter/project_collector_test.go:4]: we should follow the go-cilint style to set the import.

```
build-in
harbor 
third parth
```

**Comment by Vad1mo** [comment]
@wy65701436 any chance we can get this merged into 2.11?

**Comment by Vad1mo** [comment]
@goharbor/all-maintainers, can we get this merged? we have it now running since 1 year in prod across multiple instances and I don't see a reason not merging it.

**Review by OrlinVasilev — APPROVED** [review]

**Pull request merged**